RecoLocalCalo/HGCalRecAlgos/interface/HGCalImagingAlgo.h

```diff
@@ -96,7 +101,10 @@ HGCalImagingAlgo(const std::vector<double>& vecDeltas_in, double kappa_in, doubl
                  double fcPerEle_in,
                  const std::vector<double>& nonAgedNoises_in,
                  double noiseMip_in,
-                 VerbosityLevel the_verbosity = pERROR) : vecDeltas_(vecDeltas_in), kappa_(kappa_in),
+                 VerbosityLevel the_verbosity = pERROR) :
+        W0threshold_(W0threshold_in),
+	positionDeltaRho_c_(positionDeltaRho_c_in),
```

Comment:
inconsistent indentation

Reply:
ok. I also fixed the indentation for other few lines in the same constructor